vispy/visuals/rectangle.py

```diff
@@ -37,19 +37,26 @@ class RectangleVisual(PolygonVisual):
         Defaults to 0.
     """
     def __init__(self, pos=None, color='black', border_color=None,
-                 height=1.0, width=1.0, radius=[0., 0., 0., 0.], **kwargs):
-        super(RectangleVisual, self).__init__()
-        self.mesh.mode = 'triangle_fan'
-        self._vertices = None
-        self._pos = pos
+                 border_width=0, height=1.0, width=1.0,
+                 radius=[0., 0., 0., 0.], **kwargs):
+
+        self._center = pos
         self._color = Color(color)
         self._border_color = Color(border_color)
+        self._border_width = border_width
         self._height = height
         self._width = width
+
+        PolygonVisual.__init__(self, pos=None, color=color,
+                               border_color=border_color,
+                               border_width=border_width, **kwargs)
+
         self.radius = radius
-        self._update()
 
-    def _generate_vertices(self, pos, radius, height, width):
+        self._mesh.mode = 'triangle_fan'
+        self._update_vertices()
+
+    def _generate_vertices(self, center, radius, height, width):
```

Comment:
because "center" is more descriptive. It's the center of the rectangle, whereas "pos" is the array of points for rendering (the tri fan)